src/app/(setup)/setup/page.tsx

```diff
@@ -1,70 +1,44 @@
 'use client'
 
-import { StepTransition } from '@/components/Setup/components/StepTransition'
-import { useSetupFlow } from '@/components/Setup/context/SetupFlowContext'
-import { twMerge } from 'tailwind-merge'
+import { SetupWrapper } from '@/components/Setup/components/SetupWrapper'
+import { useSetupFlow } from '@/hooks/useSetupFlow'
+import { useSetupStore } from '@/redux/hooks'
```

Comment:
Q: why redux over zustand?

Reply:
Thuoght Q2: imo we should move a lot of the cashout / KYC state to redux too

Reply:
@Hugo0  yes ideally everything will move to redux including cashout and kyc.

chose redux cuz, it's setup is self explanatory and I've used zustand with the approach of using it by creating hooks for everything, considering the project size, i felt it would be better to avoid that, as later on other components will/should have their own hooks to handle business logic. Also for the fact that I enjoy working with redux :)